openerp/addons/base/ir/ir_cron_view.xml

```diff
@@ -6,6 +6,9 @@
             <field name="model">ir.cron</field>
             <field name="arch" type="xml">
                 <form string="Scheduled Actions">
+                    <header>
+                        <button name="method_direct_trigger" type="object" string="launch" class="oe_highlight"/> 
+                    </header>
```

Comment:
It would be nice to also add the button in the tree view as it's done in the oca module that your changes will replace. https://github.com/OCA/server-tools/blob/8.0/cron_run_manually/view/ir_cron.xml